<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettings</class>
 <widget class="QWidget" name="EditorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_4">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Autocompletion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="autocompletionEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="autocompletionCaseSensitive">
        <property name="text">
         <string>CaseSensitive</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="autocompletionReplaceWord">
        <property name="text">
         <string>Replace word</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="autocompletionShowSingle">
        <property name="text">
         <string>Show single</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="autocompletionFillups">
        <property name="text">
         <string>Fillups</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Treshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autocompletionTreshold"/>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Source</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="autocompletionSource">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Document</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>APIs</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Use single</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="autocompletionUseSingle">
          <item>
           <property name="text">
            <string>Never</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Explicit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string>Indentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="qutoIndent">
        <property name="text">
         <string>Auto indent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="backspaceUnindents">
        <property name="text">
         <string>Backspace unindents</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="indentationGuides">
        <property name="text">
         <string>Indentation guides</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useTabs">
        <property name="text">
         <string>Use tabs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tabIndents">
        <property name="text">
         <string>Tab indents</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Indentation width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="indentationWidth"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Tab width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tabWIdth"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Edge</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Column</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="edgeColumn"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="edgeType">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Background</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Folding</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="foldingStyle">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plain</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Circled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Boxed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Circled Tree</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Boxed Tree</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Wrap</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Wrap mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="wrapMode">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>By text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>By border</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Wrap indentation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="wrapIndentation">
        <item>
         <property name="text">
          <string>Fixed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Indented</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
